distsql/distsql.go

```diff
@@ -169,5 +170,35 @@ func canUseChunkRPC(ctx sessionctx.Context) bool {
 	if ctx.GetSessionVars().EnableStreaming {
 		return false
 	}
+	if !supportedAlignment() {
```

Comment:
We may check this when process starting and cache the result, instead of checking in each session.